aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -131,7 +144,12 @@ export const PayExample = () => {
         >
           <Form
             messages={{
-              required: '! This is a required field.',
+              required: (
+                <Error
+                  background="background-front"
+                  textError="This is a required field."
+                />
```

Comment:
This might seem nit picky, but since these are being used as code samples, from a style and simplicity standpoint, I'd think we are trying to avoid unnecessarily creating props. We can make this more generic and less opinionated by passing through children.
```suggestion
                <Error>This is a required field.</Error>
```

Reply:
that sounds good! Also I think Im going to move the icon to be more of grommet theme option